components/style/properties/longhand/box.mako.rs

```diff
@@ -1213,6 +1213,8 @@ ${helpers.predefined_type("scroll-snap-coordinate",
         ///
         /// Part of CSS Transform Module Level 2 and defined at
         /// [§ 13.1. 3D Transform Function](https://drafts.csswg.org/css-transforms-2/#funcdef-perspective).
+        ///
+        /// The value must be greater than zero. (Zero included.)
```

Comment:
Probably better wording. I tried to copied the definition from the spec. :unamused: